### PR TITLE
Add Java tests

### PR DESCRIPTION
**Pull request by dkurt**
* Linux and Mac Java tests (build only for Windows)
* Also changed new lines from \r\n to \n

### CODE REVIEW

**Review by azhogov — APPROVED** [review]

File: .ci/azure/linux.yml [hunk 1]
```diff
@@ -1,84 +1,101 @@
...
+      workingDirectory: $(BUILD_DIR)
+
+  - script: ninja
+    workingDirectory: $(BUILD_DIR)
+    displayName: 'Build Lin'
+
+  - script: ls -alR $(OPENVINO_REPO_DIR)/bin/
+    displayName: 'List files'
+
+  - script: java -cp ".:*" OpenVinoTestRunner
```
Comment (dkurt) [inline .ci/azure/linux.yml:97]: test failed but CI is green...

Comment (dkurt) [reply .ci/azure/linux.yml:97]: @azhogov , @ilya-lavrenov , do you know the reason why CI is passed with failed tests?

Comment (dkurt) [reply .ci/azure/linux.yml:97]: Never mind - seems like it's internal Java tests issue.

**Review by likholat — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@dkurt many thanks!

**Pull request merged**